### PR TITLE
[Draft] Add SingleStage TTAModel

### PR DESCRIPTION
**Pull request by HAOCHENYE**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: tools/test.py [hunk 1]
```diff
@@ -43,6 +43,8 @@ def parse_args():
         choices=['none', 'pytorch', 'slurm', 'mpi'],
         default='none',
         help='job launcher')
+    parser.add_argument('--tta-pipeline', default=None)
```
[the remark this answers is not in the document]
Comment (ZwwWayne) [reply tools/test.py:46]: We can implement a method in detectors like to_tta, which can convert them to their corresponding TTA model

[the remark this answers is not in the document]
Comment (HAOCHENYE) [reply tools/test.py:46]: If the config of "tta transform" and the config of "tta model" are independent of each other, and the both configuration files can be configured independently in `test.py`, then the downstream repository does not need to modify any of the config when adapting tta. 

if `test.py` only accepts `--tta`, does that mean we need to define the tta-related configuration directly in the config.

File: tools/test.py [hunk 2]
```diff
@@ -94,6 +96,15 @@ def main():
 
     cfg.load_from = args.checkpoint
 
+    if args.tta_pipeline:
+        assert args.tta_model is not None
+        cfg.merge_from_dict(Config.fromfile(args.tta_pipeline))
+        cfg.merge_from_dict(Config.fromfile(args.tta_model))
```
Comment (RangiLyu) [inline tools/test.py:102]: What will the tta-model config be like?

File: configs/_base_/datasets/coco_detection.py [hunk 3]
```diff
@@ -83,3 +83,4 @@
 #     format_only=True,
 #     ann_file=data_root + 'annotations/image_info_test-dev2017.json',
 #     outfile_prefix='./work_dirs/coco_detection/test')
+
```
Comment (BIGWangYuDong) [inline configs/_base_/datasets/coco_detection.py:86]: unnecessary change
